config/localTemplate.go

```diff
@@ -78,6 +78,31 @@ type Local struct {
 	CadaverSizeTarget uint64 `version[0]:"1073741824" version[24]:"0"`
 	CadaverDirectory  string `version[27]:""`
 
+	// Data directories
+	// HotDataDir is an alternative to DataDir for storing data that is frequently accessed
+	// if not set, resources which use HotDataDir should default to DataDir
+	HotDataDir string `version[0]:""`
```

Comment:
I think Hot/Cold have to list which files they include, so update the comment.
Additionally, config v28 was just released, so please bump its version to 29 and regenerate files.

Reply:
Maybe document the expected behavior if someone were to provide a relative paths as well. I don't think we should go out of our way to support that, the user should provide the absolute path to each of these things. Probably worth calling out though.